### ISSUE TITLE
Config directory creation error on Flatpak version

### ISSUE THREAD

**Issue by sabriunal** [flathub/org.gnome.Evolution#97]
- OS: Ubuntu 22.04.1
- Evolution 3.45.3 (by Flathub.org)) from flathub-beta

When I try to create a gmail account evolution says (in Turkish) "/home/sabri/.var/app/org.gnome.Evolution/config dizini olu?turulurken hata: B?yle bir dosya ya da dizin yok" It may translated to "Error creating directory /home/sabri/.var/app/org.gnome.Evolution/config: No such file or directory"

I couldn't find these strings in Evolution's po files. It may be related to glib or gvsf. You can see here that I have a config folder in Evolution directory.

![evolution](https://user-images.githubusercontent.com/6218679/190993889-d0593e25-7f67-41cf-8430-fd20ffeba6a0.png)

How it happened.

- I installed Evolution and tried to add a gmail account.
- I couldn't create an account due to an error. #2032 
- I removed Evolution with all data `flatpak remove --delete-data -y org.gnome.Evolution`
- I installed evolution again from flathub-beta
- I tried to add a gmail account again.

```
LC_ALL=C flatpak run --verbose org.gnome.Evolution 
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/sabri/.local/share/flatpak
F: Opening user flatpak installation at path /home/sabri/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Skipping parental controls check for app/org.gnome.Evolution/x86_64/beta since parental controls are disabled globally
F: Opening user flatpak installation at path /home/sabri/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: /var/lib/flatpak/runtime/org.gnome.Platform/x86_64/43beta/fd3b00e0ac1de421c108f6571fc76652c558a177716a079d689a670bb4e592c4/files/lib32 does not exist
F: Cleaning up unused container id 1844724026
F: Allocated instance id 1684019793
F: Add values in dir '/org/gnome/evolution/', prefix is '/org/gnome/Evolution/'
F: Add defaults in dir /org/gnome/Evolution/
F: Add locks in dir /org/gnome/Evolution/
F: writing D-Conf values to /home/sabri/.var/app/org.gnome.Evolution/config/glib-2.0/settings/keyfile
F: Allowing dri access
F: Allowing wayland access
F: Allowing x11 access
F: Allowing pulseaudio access
F: Pulseaudio user configuration file '/home/sabri/.config/pulse/client.conf': Error opening file /home/sabri/.config/pulse/client.conf: No such file or directory
F: Running 'bwrap --args 41 xdg-dbus-proxy --args=43'
F: Running 'bwrap --args 41 evolution'
Gtk-Message: 15:26:37.295: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:26:37.296: Failed to load module "canberra-gtk-module"

(evolution.bin:26): e-mail-engine-WARNING **: 15:26:37.428: Failed to add service 'Sendmail' (sendmail): No provider available for protocol ?sendmail?

(evolution.bin:26): module-rss-WARNING **: 15:26:37.438: Failed to commit RSS source: /home/sabri/.var/app/org.gnome.Evolution/config dizini olu?turulurken hata: B?yle bir dosya ya da dizin yok
Gtk-Message: 15:26:37.495: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:26:37.507: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:26:37.629: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:26:37.630: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:26:43.205: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:26:43.206: Failed to load module "canberra-gtk-module"

(evolution.bin:26): GLib-GIO-WARNING **: 15:27:48.571: Your application did not unregister from D-Bus before destruction. Consider using g_application_run().
```

Original issue: (https://gitlab.gnome.org/GNOME/evolution/-/issues/2033)

**Comment by mcrha** [comment]
Did you try the path in the sandbox, as I suggested at:  
https://gitlab.gnome.org/GNOME/evolution/-/issues/2033#note_1554554

please? The `/home/sabri/.var/app/org.gnome.Evolution/config` seems to be inaccessible for some reason, it can be wrong owner, read permissions only, maybe the whole `~/.var/app/org.gnome.Evolution` has some flags reset/unset/broken from the previous install.

Also, closing Evolution does not close the background processes it started inside its sandbox, thus something like `flatpak kill org.gnome.Evolution` may only close those background processes (`ps ax | grep evolution` should not show any processes from `/app/` path when the flatpak properly kills the background processes).

**Comment by sabriunal** [comment]
It seems fixed on 3.47.1 from flathub beta.

Thanks.
